skimage/measure/tests/test_fit.py

```diff
@@ -386,3 +386,11 @@ def residuals(self, data):
     data = np.arange(4)
     ransac(data, DummyModel, min_samples=3, residual_threshold=0.0,
            max_trials=10)
+
+
+def test_ransac_with_no_final_inliers():
+    data = np.random.rand(5, 2)
+    model, inliers = ransac(data, model_class=LineModelND, min_samples=3,
+                            residual_threshold=0, random_state=1523427)
```

Comment:
```suggestion
                                min_samples=3, residual_threshold=0,
                                random_state=1523427)
```

Reply:
@wwymak if you include the above two code suggestions, also include
```py
from skimage._shared._warnings import expected_warnings
```
in the imports :)